README.md

```diff
@@ -291,6 +291,116 @@ do this on [my personal site](https://github.com/kentcdodds/kentcdodds.com)
 which is generated at build-time on the server. Learn about rendering
 [`react` on the server][react-ssr] and [`glamor` too][glamor-ssr].
 
+### Example Style Objects
+
+Style objects can affect pseudo-classes and pseduo-elements, complex CSS
+selectors, introduce keyframe animations, and use media queries:
+
+<details>
+  <summary>pseudo-class</summary>
+  <p>
```

Comment:
This is hard to show in a github comment, but this is what it looks like:

<img width="383" alt="screen shot 2017-04-05 at 4 07 56 pm" src="https://cloud.githubusercontent.com/assets/1500684/24729144/13fc713e-1a1a-11e7-9d0d-ba3c65d0a559.png">